crates/transaction-pool/src/pool/mod.rs

```diff
@@ -242,6 +245,15 @@ where
         self.transaction_listener.lock().push(listener);
         rx
     }
+    /// Adds a new blob sidecar listener to the pool that gets notified about every new
+    /// eip4844 transaction's blob sidecar.
+    pub fn add_blob_sidecar_listener(&self) -> mpsc::Receiver<BlobTransactionSidecar> {
+        const BLOB_SIDECAR_LISTENER_BUFFER_SIZE: usize = 2048;
```

Comment:
Shouldn't this constant be extracted from the function as a general constant?

Reply:
I second that and also this should be smaller, maybe 512 since they can get quite large

Reply:
for sure. the const being added to the func, instead of outside was a reuse of the existing pattern in the same file seen [here](https://github.com/paradigmxyz/reth/blob/main/crates/transaction-pool/src/pool/mod.rs#L227) (and in the other listener funcs)

as a style thing I can take care of moving all of those out into generally defined consts.

Reply:
should be taken care of